tests/cypress/component/root-layout.rpc.cy.tsx

```diff
@@ -34,7 +33,7 @@ describe('RootLayout RPC Tests', () => {
 
   it(`redirects to arbitrum when the wallet is connected to it`, () => {
     cy.mount(
-      <ComponentTestWrapper config={createTestWagmiConfigFromVNet({ vnet: getVirtualNetwork() })} autoConnect>
+      <ComponentTestWrapper config={createTenderlyWagmiConfigFromVNet({ vnet: getVirtualNetwork() })} autoConnect>
```

Comment:
I do not like to see Tenderly directly in the tests, it should've been abstracted away. Tenderly is expansive and we might find a better solution

Reply:
It's just good practice to proxy external dependencies and abstract them. In this case this is just a name, so not real important, but the point is that we should try to provide a "neutral" interface to tenderly so it can be replaced

Reply:
The good news is, that's kinda what this PR does for a bit, as it creates a more general `test connector` in ui-kit that takes a custom transport factory as an argument. Now `createTenderlyWagmiConfigFromVNet` is merely a helper function that creates a test wagmi config with the tenderly specific transport. Maybe not perfect yet but a step in the right direction?